frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyMfaCodeHandler.java

```diff
@@ -238,15 +238,21 @@ private void processCodeSession(
 
     private void blockCodeForSessionAndResetCountIfBlockDoesNotExist(
             Session session, MFAMethodType mfaMethodType) {
```

Comment:
Aware this already existed but we could pass emailAddress directly into this method.

Reply:
Also a good point; changing now